### DISCUSSION TITLE
Discrepancy between Emporia totals and Vuegraf/Influx/Grafana totals.

### DISCUSSION THREAD

**Discussion by poorredneck** [jertel/vuegraf#106] [Q&A] [answered]
Not sure what logs to include. Please let me know what logs if any are needed.

For all intensive purposes everything seems to work fine. No big missing chunks of time on any graphs, etc. Any who the totals from Influx/Grafana are consistently off. 

This is the influx query:

`SELECT sum("usage")  / 3600 FROM "energy_usage" WHERE ("device_name" = 'House') AND $timeFilter`

vuegraf Json:

{
    "influxDb": {
        "host": "XXX.XXX.XXX.XXX",
        "port": 8086,
        "user": "XXXXXXX",
        "pass": "XXXXXXXXXXXXXXXXXX",
        "database": "vue",
        "reset": false,
        "ssl_enable": false,
        "ssl_verify": true,
		"historyDays": 0
    },
    "accounts": [
        {
            "name": "House",
            "email": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
            "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXX",
            "devices": [
                {
                    "name": "House",
                    "channels": [
                        "Pool Heater",
                        "Pool Pump",
                        "Living Room/Lights",
                        "Pool Outlet",
                        "Water Heater",
                        "Dryer",
                        "Washing Machine",
                        "Hot Tub Outside Outlets Garage",
						"Master Bedroom",
						"Dishwasher",
						"AC Heatpump",
						"Blower Motor Emer Heat",
						"Lorelei and Logan Bedrrom",
						"Microwave",
						"Kitchen Dining Outlets",
						"Refrigerator"
                    ]
                }
            ]
        }
    ]
}

![image](https://user-images.githubusercontent.com/15825016/197102605-a3d78bd2-b194-45fa-8074-35e3a8d423aa.png)
![image](https://user-images.githubusercontent.com/15825016/197102504-eb2ec24f-7551-4b43-b617-97a56c9151e9.png)

![image](https://user-images.githubusercontent.com/15825016/197102621-01055c72-17d8-4d8d-a1a6-290e8e56665a.png)
![image](https://user-images.githubusercontent.com/15825016/197102629-652cf73f-1cd6-4555-b71d-d24712e8ac21.png)

![image](https://user-images.githubusercontent.com/15825016/197102641-d20963e4-79f3-4829-9a69-0e98dd39c18b.png)

**Comment by jertel** [comment, marked as answer]
See the last paragraph, specifically the last sentence, of #57 --> Perhaps you're summing duplicate data. It could also be possible that Emporia uses a different timezone boundary for daily aggregations than your Grafana queries.

**Reply by poorredneck** [reply]
Bingo! Detailed=True fixes it, for previous time periods. Throws off "current day" metrics but that I can live with that, as those panels I only pay attention to for previous days anyways.

Thanks!
